<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConditionalBranchDialog</class>
 <widget class="QDialog" name="ConditionalBranchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_1">
      <attribute name="title">
       <string>1</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="5" column="2">
          <widget class="QSpinBox" name="timerMinSpinBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="suffix">
            <string> min</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QComboBox" name="timerSignComboBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <item>
            <property name="text">
             <string>&gt;=</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>&lt;=</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QRadioButton" name="timerRadioButton">
           <property name="text">
            <string>Timer</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">mainRadioGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QRadioButton" name="selfSwitchRadionButton">
           <property name="text">
            <string>Self Switch</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">mainRadioGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QRadioButton" name="switchRadioButton">
           <property name="text">
            <string>Switch</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">mainRadioGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="6" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="variableRadioButton">
           <property name="text">
            <string>Variable</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">mainRadioGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="5" column="3">
          <widget class="QSpinBox" name="timerSecSpinBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="suffix">
            <string> sec</string>
           </property>
           <property name="maximum">
            <number>59</number>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QRadioButton" name="variableConstantRadioButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Constant</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">variableRadioGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QRadioButton" name="variableVariableRadionButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Variable</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">variableRadioGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="SelectorButton" name="switchButton">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="SelectorButton" name="variableButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QComboBox" name="switchComboBox">
           <item>
            <property name="text">
             <string>ON</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>OFF</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QComboBox" name="variableSignComboBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <item>
            <property name="text">
             <string>==</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>&gt;=</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>&lt;=</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>&gt;</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>&lt;</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>!=</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QSpinBox" name="variableConstantSpinBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimum">
            <number>-99999999</number>
           </property>
           <property name="maximum">
            <number>99999999</number>
           </property>
          </widget>
         </item>
         <item row="3" column="2" colspan="2">
          <widget class="SelectorButton" name="variableVariableButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="selfSwitchComboBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <item>
            <property name="text">
             <string>A</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>B</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>C</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>D</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QComboBox" name="selfSwitchToggleComboBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <item>
            <property name="text">
             <string>ON</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>OFF</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>2</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="4" column="1">
          <widget class="QRadioButton" name="actorSkillRadioButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Skill</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">actorRadioGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QRadioButton" name="actorInPartyRadioButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>In the party</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">actorRadioGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QRadioButton" name="actorWeaponRadioButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Weapon</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">actorRadioGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QRadioButton" name="actorArmorRadioButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Armor</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">actorRadioGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QRadioButton" name="actorClassRadioButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Class</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">actorRadioGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="SelectorButton" name="actorSkillButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QRadioButton" name="actorNameRadioButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Name</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">actorRadioGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="SelectorButton" name="actorButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="6" column="2">
          <widget class="SelectorButton" name="actorArmorButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="SelectorButton" name="actorClassButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QRadioButton" name="actorRadioButton">
           <property name="text">
            <string>Actor</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">mainRadioGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLineEdit" name="actorNameLineEdit">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="SelectorButton" name="actorWeaponButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="7" column="1">
          <widget class="QRadioButton" name="actorStateRadioButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>State</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">actorRadioGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="7" column="2">
          <widget class="SelectorButton" name="actorStateButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>3</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="2" column="1">
          <widget class="QRadioButton" name="enemyStateRadioButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>State</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">enemyRadioGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QComboBox" name="characterDirectionComboBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QComboBox" name="vehicleComboBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="characterLabel">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>is facing</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="SelectorButton" name="enemyStateButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QLabel" name="vehicleLabel">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>driven</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="enemyComboBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="characterComboBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QRadioButton" name="vehicleRadioButton">
           <property name="text">
            <string>Vehicle</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">mainRadioGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QRadioButton" name="characterRadioButton">
           <property name="text">
            <string>Character</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">mainRadioGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QRadioButton" name="enemyAppearedRadioButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Appeared</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">enemyRadioGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QRadioButton" name="enemyRadioButton">
           <property name="text">
            <string>Enemy</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">mainRadioGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="6" column="0">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>4</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="2" column="0">
          <widget class="QRadioButton" name="weaponRadioButton">
           <property name="text">
            <string>Weapon</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">mainRadioGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QSpinBox" name="goldSpinBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="maximum">
            <number>9999999</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QRadioButton" name="goldRadioButton">
           <property name="text">
            <string>Gold</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">mainRadioGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="itemRadioButton">
           <property name="text">
            <string>Item</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">mainRadioGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QComboBox" name="buttonComboBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="goldSignComboBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="6" column="2">
          <widget class="QLabel" name="buttonLabel">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>pressed</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1" colspan="2">
          <widget class="QLineEdit" name="scriptLineEdit">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="SelectorButton" name="itemButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="2">
          <widget class="SelectorButton" name="weaponButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1" colspan="2">
          <widget class="SelectorButton" name="armorButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1" colspan="2">
          <widget class="QCheckBox" name="weaponCheckBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Include Equipment</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1" colspan="2">
          <widget class="QCheckBox" name="armorCheckBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Include Equipment</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QRadioButton" name="armorRadioButton">
           <property name="text">
            <string>Armor</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">mainRadioGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QRadioButton" name="buttonRadioButton">
           <property name="text">
            <string>Button</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">mainRadioGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QRadioButton" name="scriptRadioButton">
           <property name="text">
            <string>Script</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">mainRadioGroup</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="elseBranchCheckBox">
     <property name="text">
      <string>Create Else Branch</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SelectorButton</class>
   <extends>QPushButton</extends>
   <header>selector_button.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConditionalBranchDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConditionalBranchDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="mainRadioGroup"/>
  <buttongroup name="actorRadioGroup"/>
  <buttongroup name="variableRadioGroup"/>
  <buttongroup name="enemyRadioGroup"/>
 </buttongroups>
</ui>
